nuxeo-services/nuxeo-platform-convert/src/test/java/org/nuxeo/ecm/platform/convert/tests/TestZip2HtmlConverter.java

```diff
@@ -59,26 +59,8 @@ public void testConvertToMimeType(String mimeType) throws IOException {
         assertEquals(3, blobs.size());
 
         String content = blobs.get(0).getString();
-        String zipList = "<html>" +
-                "<body>" +
-                "<h1>hello.zip</h1>" +
-                "<ul>" +
-                "<li><a href=\"hello.xml\">hello.xml</a></li>" +
-                "<li><a href=\"hello.txt\">hello.txt</a></li>" +
-                "</ul>" +
-                "</body>" +
-                "</html>";
-        String zipListAlternative = "<html>" +
-                "<body>" +
-                "<h1>hello.zip</h1>" +
-                "<ul>" +
-                "<li><a href=\"hello.xml\">hello.xml</a></li>" +
-                "<li><a href=\"hello.txt\">hello.txt</a></li>" +
-                "</ul>" +
-                "</body>" +
-                "</html>";
-
-        assertTrue(zipList.equals(content) || zipListAlternative.equals(content));
+        assertTrue(content.contains("<li><a href=\"hello.xml\">hello.xml</a></li>") &&
+                content.contains("<li><a href=\"hello.txt\">hello.txt</a></li>"));
```

Comment:
Could you convert this to two `assertTrue` ?

Reply:
Yes, of course. Thanks, it improves readability 🙂

Reply:
Do you think that there can be a problem by keeping such a structure for comparison? (maintaining " and quotes)